conanfile.py

```diff
@@ -100,6 +100,10 @@ def _marketplace_root(self):
     def _digital_factory_url(self):
         return "https://digitalfactory-staging.ultimaker.com" if self._staging else "https://digitalfactory.ultimaker.com"
 
+    @property
+    def _cura_latest_url(self):
+        return "https://raw.githubusercontent.com/Ultimaker/Uranium/CURA-9272_semver_postfix/tests/latest.json"
```

Comment:
why not use `ApplicationMetadata.CuraLatestURL` here?
```suggestion
        return ApplicationMetadata.CuraLatestURL
```